src/common/api.ts

```diff
@@ -0,0 +1,45 @@
+import * as dns from '../dns'
```

Comment:
Why move all of this to `common` instead of `core`? Especially with our other `did` stuff still in `core`? 

I actually split out a lot of `did` stuff into `core/did` in the `keys` PR that I have up right now: https://github.com/fission-suite/ts-sdk/pull/50

Reply:
I moved it to `./common` because it's not so much to do with DIDs, it's just a lookup for a DID, specifically related to Fission APIs.

Your code looks good 👍

Reply:
okay cool makes sense :+1: